test/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tests/ApplicationConsumingPrecompiledViews.cs

```diff
@@ -65,9 +75,14 @@ private void CreateClassLibraryPackage()
             {
                 RestoreProject(ClassLibraryPath);
                 ExecuteForClassLibrary(Command.CreateDotNet("build", new[] { ClassLibraryPath, "-c", "Release" }));
-                ExecuteForClassLibrary(Command.CreateDotNet(
-                    "razor-precompile",
-                    GetPrecompileArguments("net451")));
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
It's still kinda weird because it doesn't have precompiled views in one case, but it works.